financial-templates-lib/test/UniswapPriceFeed.js

```diff
@@ -32,11 +33,26 @@ contract("UniswapPriceFeed.js", function(accounts) {
     assert.equal(uniswapPriceFeed.getCurrentPrice().toString(), toWei("0.25"));
   });
 
+  it("Selects most recent price in same block", async function() {
+    // Just use current system time because the time doesn't matter.
+    const time = Math.round(new Date().getTime() / 1000);
+
+    const transactions = [
+      uniswapMock.contract.methods.setPrice(toWei("1"), toWei("1")),
+      uniswapMock.contract.methods.setPrice(toWei("2"), toWei("1")),
+      uniswapMock.contract.methods.setPrice(toWei("4"), toWei("1"))
+    ];
+
+    await mineTransactionsAtTime(web3, transactions, time, owner)
```

Comment:
A test to show that this only advances the block number by one block would make this clearer for me. Something to show that after the call the block number has only progressed by `addn(1)` from the block number before, even though 3 transactions where included.

Reply:
Agreed, SGTM.

Reply:
Done.